composer.json

```
@@ -0,0 +1,28 @@
+{
+    "name": "benjaminhu/tsheets-api-phplib",
+    "description": "PHP Helper Library for TSheets API",
+    "type": "library",
+    "homepage": "https://github.com/TSheetsTeam/api_phplib",
+    "keywords": [
+        "library",
+        "php",
+        "rest",
+        "tsheets",
+        "tsheets quickbooks",
+        "tsheets api"
+    ],
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Benjamin Simon"
+        }
+    ],
```

If we do something like this for Authors we can attribute everyone who's contributed to the project:
```suggestion
    "authors": [
        {
            "name": "TSheets API PHP Library contributors",
            "homepage": "https://github.com/TSheetsTeam/api_phplib/graphs/contributors"
        }
    ],
```